stylesheet/style.css

```
@@ -0,0 +1,208 @@
+@import "reset.css";
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+* {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+body {
+  font-family:'Roboto', sans-serif;
```

```suggestion
  font-family: 'Roboto', sans-serif;
```

забыла пробел после двоеточия